ofl/mplus1/METADATA.pb

```diff
@@ -0,0 +1,24 @@
+name: "Mplus 1"
```

Comment:
This should match the naming used already, eg https://fonts.google.com/specimen/M+PLUS+Rounded+1c

Reply:
Ah! For some reason I hadn't realized there was an extant MPLUS. I'll match it.